src/assets/css/main.css

```diff
@@ -0,0 +1,9 @@
+.navbar {
+  padding: 0.7rem 2rem;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+  background: #333;
+  opacity: 0.9;
+  color: #fff; }
```

Comment:
Why aren't you using variables for the colors in this file? Also,  why do you have a navbar class in this file and in the _navbar.scss?